.github/scripts/run_sync-tools_and_sync-workflows.sh

```diff
@@ -65,8 +65,9 @@ conda run \
 # Install setuptools from pip
 echo "## DEBUG Forcing installation of setuptools to less than version 58 which seems to be causing some errors" 1>&2
 conda run \
- --name "${CONDA_ENV_NAME}" \
- bash -c "pip install setuptools<58"
+  --name "${CONDA_ENV_NAME}" \
+  bash -c 'pip install setuptools<58'
```

Comment:
The command responsible for triggering is:
```
conda run ['bash', '-c', 'pip install setuptools<58']
```

But needs to read:
```
conda run ['bash', '-c', 'pip install "setuptools<58"']
```

So the quoting needed here is:
```
bash -c 'pip install "setuptools<58"'
```